specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/examples/ReservationRecommendationsList.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "api-version": "2018-03-31",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Consumption/reservationRecommendations/reservationRecommendations1",
+            "name": "reservationRecommendations1",
+            "type": "Microsoft.Consumption/reservationRecommendations",
+            "sku": "Standard_DS1_v2",
+            "location": "northeurope",
+            "properties": {
+              "lookBackPeriod": "Last7Days",
+              "meterId": "00000000-0000-0000-0000-000000000000",
+              "term": "P1Y",
+              "costWithNoReservedInstances": 0.0,
+              "recommendedQuantity": 1,
+              "totalCostWithReservedInstances": 0.0,
+              "netSavings": 4.634521202630137,
+              "firstUsageDate": "2018-03-06T00:00:00Z ",
```

Comment:
Another extra whitespace, looks like there's on line 42 as well.  Please scrub the remaining example files for this error.

Reply:
Done! I have verified the other files as well.